Source/JavaScriptCore/dfg/DFGJITCode.cpp

```diff
@@ -51,6 +52,14 @@ JITData::JITData(const JITCode& jitCode, ExitVector&& exits)
             at(i) = &stubInfo;
             break;
         }
+        case LinkerIR::Type::CallLinkInfo: {
+            unsigned index = bitwise_cast<uintptr_t>(entry.pointer());
+            const UnlinkedCallLinkInfo& unlinkedCallLinkInfo = jitCode.m_unlinkedCallLinkInfos[index];
+            OptimizingCallLinkInfo& callLinkInfo = m_callLinkInfos[index];
+            callLinkInfo.initializeFromDFGUnlinkedCallLinkInfo(vm, unlinkedCallLinkInfo);
```

Comment:
Maybe I'm not understanding how this is supposed to be initialized, but could this be a placement new instead?

Reply:
Because it is already initialized, it is not great that we re-initialize the already initialized memory region (UB).